docs/pages/guides/using-firebase.md

```diff
@@ -46,215 +76,191 @@ const firebaseConfig = {
+For iOS, React Native Firebase requires [altering CocoaPods to use frameworks](https://rnfirebase.io/#altering-cocoapods-to-use-frameworks). The [`expo-build-properties`](/versions/latest/sdk/build-properties/) plugin allows you to override the default native build properties and configure to use frameworks.
 
-Here's is an example of storing a document named "mario" inside of a collection named "characters" in Firestore:
+Start by installing the `expo-build-properties` plugin:
 
-```javascript
-import { initializeApp } from 'firebase/app';
-import { getFirestore, setDoc, doc } from 'firebase/firestore';
+<Terminal cmd={["$ npx expo install expo-build-properties"]} />
 
-const firebaseConfig = { ... }  // apiKey, authDomain, etc. (see above)
+Then, update the config plugin array in **app.json** or **app.config.js**:
 
-let myApp = initializeApp(firebaseConfig);
+```js
+{
+  "plugins": [
+      "@react-native-firebase/app",
+      [
+         "expo-build-properties",
+        {
+          "ios": {
+            "useFrameworks": "static"
+          }
+        }
+      ]
+    ]
+}
+```
 
-const firestore = getFirestore(myApp);
+#### Step 6: Run the project
 
-await setDoc(doc(firestore, "characters", "mario"), {
-  employment: "plumber",
-  outfitColor: "red",
-  specialAttack: "fireball"
-});
-```
+If you are using **[EAS Build](/build/introduction/), you can create and install a development build** on your devices. You do not need to run the project locally before creating the development build. For more information on creating a development build, see [Getting Started with Development builds](/development/getting-started/#creating-and-installing-your-first-development-build).
 
-This sample was borrowed and edited from [this forum post](https://forums.expo.dev/t/open-when-an-expo-firebase-firestore-platform/4126/29).
+To run the project locally:
 
-## Recording events with Analytics
+- You need Android Studio, and Xcode installed and configured on your computer.
+- Then, you can run the project using `npx expo run:android` or `npx expo run:ios` command.
 
-In order to record analytics events, the Expo Firebase Core and Analytics packages needs to be installed.
+If a particular React Native Firebase module requires custom native configuration steps, you must add it as a `plugin` to **app.json** or **app.config.js**. Then, to run the project locally, you will have to run the `npx expo prebuild --clean` command to apply the native changes before the `npx expo run` commands.
 
-<Terminal cmd={['$ npx expo install expo-firebase-analytics']} />
+### Analytics
```

Comment:
Someone who has gone through the steps above (setting up RN Firebase and a dev client) is in a great position to avoid using expo-firebase-analytics and use RN Firebase's solution instead. I'd be in favor of eliminating this section.

Reply:
I agree with you on this especially after the updates were merged last week.